requirements-dev.txt

```diff
@@ -17,3 +17,6 @@ black
 mkdocs
 mkdocs-material
 markdown-include
+
+# Setup
```

Comment:
I'm not sure if it's the fix. Let's see in another PR

Reply:
Ok, I'll wait with merging. No use having it there if it was just luck.

Reply:
Looks rather promising!